tests/test_static.py

```diff
@@ -106,11 +106,15 @@ def testBadChecksum(self):
         self.assertEqual(res, False)
 
     def testDMM2DDD(self):

```

Comment:
as per comment above, maybe add test for missing dp?

Reply:
done